scripts/RNA_analysis/make_complexity_plots.py

```diff
@@ -27,12 +28,13 @@
     extra_arg = config['sbatch']['extra_arg']
     tools   = config['custom_algorithms']['RNA-seq analysis']
     bam     = tools['bamtools']+'/'+tools['bamtools_version']
+    bed     = tools['BEDTools']+'/'+tools['BEDTools_version']
     preseq  = tools['Preseq']+'/'+tools['Preseq_version']
 except:
-    print 'ERROR: problem loading samtools version from config file'
+    print 'ERROR: problem loading tools version from config file'
 
 
-f=open("complexity_"+name+".sh",'w')
+f=open("complexity_"+name+".sh",'wa')
```

Comment:
@jun-wan  much better to use 
` with open(file, 'wa') as f:`
it handles for you the closing of the file.